### PR TITLE
[Event Hubs Client] Formatting Pass

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The purpose of these changes is to format code to apply project conventions for consistency and in some cases, update member names and documentation to better convey context.

**Note:** These changes are intended to be superficial; no change to existing behavior should have taken place.

# Last Upstream Rebase

Thursday, February 2, 10:45am (EST)

### CODE REVIEW

File: sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientTests.cs [hunk 1]
```diff
@@ -131,27 +131,27 @@ public void ConstructorsTranslateOptionsForTheBaseClass()
...
-               ConnectionOptions = new EventHubConnectionOptions { TransportType = EventHubsTransportType.AmqpWebSockets },
-               RetryOptions = new EventHubsRetryOptions { MaximumRetries = 99 },
-               Identifier = "OMG, HAI!",
-               MaximumWaitTime = TimeSpan.FromDays(54),
-               TrackLastEnqueuedEventProperties = true,
-               PrefetchCount = 5,
-               PrefetchSizeInBytes = 500,
-               LoadBalancingUpdateInterval = TimeSpan.FromDays(65),
-               PartitionOwnershipExpirationInterval = TimeSpan.FromMilliseconds(65)
+                ConnectionOptions = new EventHubConnectionOptions { TransportType = EventHubsTransportType.AmqpWebSockets },
```
Comment (conniey) [inline sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientTests.cs:146]: How were these 3 space indents be missed!? 😢

**Review by conniey — APPROVED** [review]

File: sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientTests.cs (hunk 1 above)
Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientTests.cs:146]: Counting is hard?

**Pull request merged**